air-quality-backend/scripts/run_forecast_etl.py

```diff
@@ -33,6 +34,13 @@ def main():
     logging.info("Persisting forecast data")
     insert_data(transformed_forecast_data)
 
+    logging.info(extracted_forecast_data._single_level_data.encoding)
+    logging.info(extracted_forecast_data._multi_level_data.encoding)
+
+    keep_files = os.environ.get("STORE_GRIB_FILES", "False") == "True"
```

Comment:
Out of interest did we check if there's a way to do this closer to the file creation? If for example we were to include this functionality in the forecast_dao.py fetch_cams_data call itself?
Doing it there would limit the amount of the codebase that needs to know aboutt his stuff, and we wouldn't need to include it in future if we need to call CAMs again (like we do for OpenAQ).